cmd/osmosisd/cmd/airdrop.history

```diff
@@ -0,0 +1,253 @@
+/*
+
+Preserved for historical purposes
+package cmd
```

Comment:
Can we rename this file `airdrop.go.history`

Reply:
awesome